### PR TITLE
mining gold

### PR DESCRIPTION
**Pull request by Mistake91**

### CODE REVIEW

File: pages/goldmine.js [hunk 1]
```diff
@@ -3,14 +3,29 @@ import styled from "styled-components";
...
 import { Workstation, Character } from "../components/homeStyles";
 
-export default function Goldmine() {
+export default function Goldmine({ materials, setMaterials }) {
   const [characterPosition, setCharacterPositon] = useState({
     row: 9,
     column: 2,
   });
 
+  const [isStopButtonVIsible, setIsStopButtonVIsible] = useState(false);
```
Comment (Long-HaHoang) [inline pages/goldmine.js:12]: You have typo in your state 
You can use `F2` to rename it.

File: pages/goldmine.js [hunk 2]
```diff
@@ -3,14 +3,29 @@ import styled from "styled-components";
...
+
   return (
     <>
+      {isStopButtonVIsible && (
```
Comment (Long-HaHoang) [inline pages/goldmine.js:16]: typo

File: pages/goldmine.js [hunk 3]
```diff
@@ -41,8 +56,22 @@ export default function Goldmine() {
 
   function positionHandler(row, column) {
     setCharacterPositon({ row, column });
+    setIsStopButtonVIsible(true);
```
Comment (Long-HaHoang) [inline pages/goldmine.js:59]: typo

**Review by Long-HaHoang — APPROVED** [review]
Besides the typo in your state it's similar to your other mines 👍

**Comment by Mistake91** [comment]
Hey Long, thanks, i will fix these typos 👍

**Pull request merged**